packages/init/src/index.ts

```diff
@@ -57,7 +57,7 @@ async function init() {
     "JfTfmHaATChc4xueS7RcCBsixI71dJIJ",
     name,
     {
-      userId: user.shortId,
+      userId: user.profile?.shortId || user.info.userId,
```

Comment:
I wouldn't build the tracker at all if I don't have the short ID to identify the user.
We can't use the userID for sure, or we would end up with events attributed to the wrong user.

Reply:
okay, so tracker is going to have to be optional later on

Reply:
done